.github/workflows/ci.yaml

```diff
@@ -1,6 +1,9 @@
 name: Continuous Integration
```

Comment:
Rather than hijacking the existing CI pipeline, we should have a new workflow called like "Deploy" (`deploy.yaml`) that only runs on the main branch

That way PRs will still have linting run on them

Reply:
I'll remove the listing from this one then